FAQ.md

```diff
@@ -0,0 +1,40 @@
+# FAQ
+
+## Can `xdotool` work with hexadecimal window IDs?
+
+Yes. You can easily check this, by using `wmctrl` as
+an alternative way to obtain a window ID in hexadecimal.
+
+This will show the window ID in hexadecimal of
+a window with title `My dialog title`:
+
+```
+wmctrl -l | egrep "My dialog title" | cut -f 1 -d ' '
+```
+
+I got the value of `0x02800003` displayed on my screen, 
+but you will probably have a different value.
+
+Feeding that hexadecimal value to `xdotool` does activate the window:
+
+```
+xdotool windowactivate $(wmctrl -l | egrep "My dialog title" | cut -f 1 -d ' ')
+```
+
+## How to let `xdotool` work with Qt applications?
+
+`xdotool` can unsually extract the ID of a Qt application 
```

Comment:
Agreed here; sometimes xdotool doesn't correctly select the application's window. I'd love to fix this since I think it's a bug, but this is a great faq entry. <3